### ISSUE TITLE
Add service provider to push image from inside the application

### ISSUE THREAD

**Issue by Goopil** [UniSharp/laravel-filemanager#348]
Hello there, Thanks for your great work ! 

I've checked a bit the code you have done so far and didn't see any way to push image from say a seeder is there any plan to add id ? 
that would be very helpful.

**Comment by g0110280** [comment]
@Goopil What does "push image" mean?

**Comment by Goopil** [comment]
@g0110280 let me rephrase it. I wanted to add  related images and files in seeder. But the way  i see it. i can't without making a call to the upload endpoint ?

**Comment by g0110280** [comment]
It cannot be accomplished for now. A lot of refactor should be done to support this feature. However, I will add this as a milestone of `v2.0`.
